clients/gtest/gerc_gtest.cpp

```diff
@@ -0,0 +1,144 @@
+/* ************************************************************************
```

Comment:
Since ger/gerc/geru are basically the same function, would it be better to have a single `ger_gtest.cpp` rather than splitting it up into three?

Reply:
I did that and it was so ugly as the 3 names need to be switched between it was much simpler making 3 test instantiations.   I think if we refactor the test instantiation class this may be doable in the future, but can likely use one class for all functions.